src/Utils.ts

```diff
@@ -49,7 +49,7 @@ const Utils = {
      * Is the app used in the context of the AEM Page editor.
      */
     isInEditor(): boolean {
-        return [ AEM_MODE.EDIT, AEM_MODE.PREVIEW ].includes(getWCMMode());
+        return [ AEM_MODE.EDIT, AEM_MODE.PREVIEW ].includes(getWCMMode()) || AuthoringUtils.isRemoteApp();
```

Comment:
Please deprecate this function and change usages to use the function from ModelManager itself.

Reply:
+1, let's keep this functionality in single place